respondd.service

```diff
@@ -1,11 +1,15 @@
+
 [Unit]
 Description=Respondd
 After=network.target
 
 [Service]
 ExecStart=/opt/mesh-announce/respondd.py -d /opt/mesh-announce/providers -i <your-clientbridge-if> -i <your-mesh-vpn-if> -b <your-batman-if> -m <mesh ipv4 address>
 Restart=always
+WorkingDirectory=/opt/mesh-announce
 Environment=PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+# User=mesh-announce
+# AmbientCapabilities=CAP_NET_BIND_SERVICE
```

Comment:
This needs some documentation, anywhere. If not in the main-readme, then in a 'drop priviliges'-readme.

Reply:
Yes, that needs to be documented. But I'm not so sure what the best general approach would be. 

Here is how i do it at the moment: https://github.com/Freifunk-Rhein-Neckar/salt/blob/master/mesh-announce/init.sls